src/CodeGen_C.cpp

```diff
@@ -122,43 +123,49 @@ const string globals =
     "inline float inf_f32() {return INFINITY;}\n"
     "inline bool is_nan_f32(float x) {return x != x;}\n"
     "inline bool is_nan_f64(double x) {return x != x;}\n"
-    "inline float float_from_bits(uint32_t bits) {\n"
-    " union {\n"
-    "  uint32_t as_uint;\n"
-    "  float as_float;\n"
-    " } u;\n"
-    " u.as_uint = bits;\n"
-    " return u.as_float;\n"
-    "}\n"
+    "template<typename A, typename B> A reinterpret(B b) {A a; memcpy(&a, &b, sizeof(a)); return a;}\n"
```

Comment:
It would show up in the generated C output... I don't think we have c++11 support there